addons/Pouches/Pouches.lua

```diff
@@ -73,9 +86,13 @@ windower.register_event('addon command', function(...)
             end
         end
         if item.count > 0 then
-            log('Found '..item.count..' '..item.name..'. Commencing Use.')
-            log('You may simply type /heal to stop.')
-            use_item()
+            if inventory_info.current < inventory_info.max then 
```

Comment:
I missed that you use it here as well. You could place the check in a function:
```lua
local check_space = function()
    local info = windower.ffxi.get_bag_info(0)
    return info.count < info.max
end
```
Then you can just use it wherever.